#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
uint8 front_pressure_raw
uint8 rear_pressure_raw
uint8 front_pressure_target
uint8 rear_pressure_target
bool brake_light
bool ignition
bool act_out_rly
bool act_in_rly
bool delayed_ignition
bool park_brake
uint8 system_state